R/Argos.r

```diff
@@ -0,0 +1,23 @@
+#'Run Argos, create R markdown HTML
+#'
+#'
+#'@export
+
+Argos <- function(){
+    
+    check.packages('ggplot2')
```

Comment:
I suggest using this function to reduce the code by using a data structure such as a vector.
```r
pkgs <- c('dplyr', 'markdown', ...)
check.packages(pkgs)
```
The above code is a pseudo code.